pkg/ssh/ssh.go

```diff
@@ -0,0 +1,514 @@
+/*
```

Comment:
I don't think this was correctly rebased.

Reply:
Fixed.